### PR TITLE
[zh-cn]: update the translation of `Fullscreen` API

### PR DESCRIPTION
**Pull request by tianyeeT**

### CODE REVIEW

File: files/zh-cn/web/api/fullscreen_api/index.md [hunk 1]
```diff
@@ -1,139 +1,107 @@
...
 
-全屏 API 在 {{DOMxRef("Document")}} 和 {{DOMxRef("Element")}} 接口中增加了一些方法，可用于允许打开关闭全屏模式。
+Fullscreen API 向 {{DOMxRef("Document")}} 和 {{DOMxRef("Element")}} 接口添加了方法，以允许打开和关闭全屏模式。
 
-### Document 中的方法
+### Document 接口上的实例方法
 
 - {{DOMxRef("Document.exitFullscreen()")}}
-  - : 用于请求从全屏模式切换到窗口模式，会返回一个 {{jsxref("Promise")}}，会在全屏模式完全关闭的时候被置为 resolved 状态。
+  - : 请求 {{Glossary("user agent")}} 从全屏模式切换回窗口模式。返回一个 {{jsxref("Promise")}}，在全屏模式完全关闭后解决。
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/fullscreen_api/index.md:25]:
```suggestion
  - : 请求 {{Glossary("user agent")}} 从全屏模式切换回窗口模式。返回一个 {{jsxref("Promise")}}，在全屏模式完全关闭后被兑现。
```

File: files/zh-cn/web/api/fullscreen_api/index.md [hunk 2]
```diff
@@ -1,139 +1,107 @@
...
 
-### Element 中的方法
+### Element 接口上的实例方法
 
 - {{DOMxRef("Element.requestFullscreen()")}}
-  - : 请求浏览器（user agent）将特定元素（甚至延伸到它的后代元素）置为全屏模式，隐去屏幕上的浏览器所有 UI 元素，以及其他应用。返回一个 {{jsxref("Promise")}}，并会在全屏模式被激活的时候变成 resolved 状态。
-
-## 属性
+  - : 请求用户代理将指定元素（及其后代）置于全屏模式，移除所有浏览器的用户界面元素以及所有其他应用程序。返回一个 {{jsxref("Promise")}}，在全屏模式激活后解决。
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/fullscreen_api/index.md:30]:
```suggestion
  - : 请求用户代理将指定元素（及其后代）置于全屏模式，移除所有浏览器的用户界面元素以及所有其他应用程序。返回一个 {{jsxref("Promise")}}，在全屏模式激活后被兑现。
```

File: files/zh-cn/web/api/fullscreen_api/index.md [hunk 3]
```diff
@@ -1,139 +1,107 @@
...
 
-_{{DOMxRef("Document")}} 提供了可以用于判断是否支持和启用全屏模式的属性，也能得知现在是否处在全屏模式、哪一个元素在使用屏幕等信息。_
+## 实例属性
 
-- {{DOMxRef("DocumentOrShadowRoot.fullscreenElement")}}
-  - : `fullscreenElement` 属性提供了当前在 DOM（或者 shadow DOM）里被展示为全屏模式的 {{DOMxRef("Element")}}，如果这个值为 `null`，文档不处于全屏模式。
+- {{DOMxRef("Document.fullscreenElement")}} / {{DOMxRef("ShadowRoot.fullscreenElement")}}
+  - : `fullscreenElement` 属性告诉你当前在 DOM（或 shadow DOM）中以全屏模式显示的 {{DOMxRef("Element")}}。如果为 `null`，则文档（或 shadow DOM）不在全屏模式。
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/fullscreen_api/index.md:35]:
```suggestion
  - : `fullscreenElement` 属性告诉你当前在 DOM（或影子 DOM）中以全屏模式显示的 {{DOMxRef("Element")}}。如果为 `null`，则文档（或影子 DOM）不在全屏模式。
```

File: files/zh-cn/web/api/fullscreen_api/index.md [hunk 4]
```diff
@@ -1,139 +1,107 @@
...
 
 ### 废弃属性
 
 - {{DOMxRef("Document.fullscreen")}} {{Deprecated_Inline}}
 
-  - : 一个布尔值，文档内任意一个元素以全屏模式程序时该值为`true`，否则为 `false`。
+  - : 一个布尔值，如果文档有一个当前正在以全屏模式显示的元素，则为 `true`；否则返回 `false`。
 
-    > **备注：** 请改用 {{DOMxRef("ShadowRoot")}} 或 {{DOMxRef("Document")}} 上的 {{DOMxRef("Document.fullscreenElement", "fullscreenElement")}} 属性；如果它不是 null，则它是就是当前在全屏模式下显示的元素。
+    > **备注：** 使用 {{DOMxRef("Document.fullscreenElement", "fullscreenElement")}} 属性在 {{DOMxRef("Document")}} 或 {{DOMxRef("ShadowRoot")}} 上；如果它不为 `null`，则它是一个当前正在以全屏模式显示的 {{DOMxRef("Element")}}。
```
Comment (PassionPenguin) [inline files/zh-cn/web/api/fullscreen_api/index.md:45]: 奇怪的语序？

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**